exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -748,6 +771,22 @@ public ParquetTableMetadataDirs(List<String> directories) {
       return directories;
     }
 
+    /** If directories list contains relative paths, update it to absolute ones
```

Comment:
Explanation of why we do this? Do we store data in relative form, but convert it to absolute form internally?